src/APIFunctions/2DPrinting.js

```diff
@@ -48,9 +49,10 @@ export function parseRange(pages, maxPages) {
  */
 export async function printPage(data) {
   let status = new PrintApiResponse();
-  await axios.post('/api/print/submit', data).catch(() => {
-    status.error = true;
-  });
+  await axios.post(RPC_API_URL + '/SceRpcApi/Printer/sendPrintRequest', data)
```

Comment:
Makes sense since most of these endpoints don't really do anything at the moment.